server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -718,6 +718,68 @@ In order to do that: `default=true`
 
 The web-admin server will return `404` response code when the default identity can not be found.
 
+=== Creating a user identity
```

Comment:
```suggestion
=== Provisioning a JMAP user-set identity

Reply:
I'm not sure,
the POST API is just a create a new entry

Reply:
Just another paraphrase explains that using create api to provision identity for users. Anyway no big deal for this.